### PR TITLE
HDzero preset update

### PR DESCRIPTION
**Pull request by sugaarK**
added support for the full range of vtxs from HDzero

this update allows all vtx from HDzero to be setup from this preset
unknown

### CODE REVIEW

File: presets/4.3/vtx/HDZero.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+#$ TITLE: HDZero VTXs
 #$ FIRMWARE_VERSION: 4.2
 #$ FIRMWARE_VERSION: 4.3
 #$ CATEGORY: VTX
 #$ STATUS: COMMUNITY
-#$ KEYWORDS:  vtx, vtx table, HDzero, divimath, shark bite, SA 2.1
+#$ KEYWORDS:  vtx, vtx table, HDzero, divimath, shark bite, SA 2.1, digital, whoop, whoop lite, race v1, race v2, freestyle, tx5m.1
 #$ AUTHOR: sugarK
-#$ DESCRIPTION: VTX table for HDZero race 2
+#$ DESCRIPTION: VTX tables and Canvas mode setup for all the HDzero digital VTXs
```
Comment (KarateBrot) [inline presets/4.3/vtx/HDZero.txt:8]:
```suggestion

#$ PARSER: MARKED

#$ DESCRIPTION:
#$ DESCRIPTION: <img src="https://static.wixstatic.com/media/967e02_82348b3d176249aea0926d07a2de0257~mv2_d_5559_3125_s_4_2.png" width="250px" style="margin-left: auto; margin-right: auto; display: block;"/>
#$ DESCRIPTION:
#$ DESCRIPTION: VTX tables and Canvas mode setup for all the HDzero digital VTXs
#$ DESCRIPTION: ------------
```

**Review by KarateBrot — CHANGES_REQUESTED** [review]

File: presets/4.3/vtx/HDZero.txt [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+#$ PARSER: MARKED
+
+#$ DESCRIPTION:
+#$ DESCRIPTION: <img src="https://static.wixstatic.com/media/967e02_82348b3d176249aea0926d07a2de0257~mv2_d_5559_3125_s_4_2.png" width="250px" style="margin-left: auto; margin-right: auto; display: block;"/>
+#$ DESCRIPTION:
+#$ DESCRIPTION: VTX tables and Canvas mode setup for all the HDzero digital VTXs
+#$ DESCRIPTION: ------------
 #$ DESCRIPTION: The information provided on this preset is for educational and entertainment purposes only. Betaflight makes no representations as to the safety or legality of the use of any information provided herein. End users assume all responsibility and liability for ensuring they are complying with all relevant laws and regulations.
 #$ DESCRIPTION: ----------
 #$ DESCRIPTION: Using the VTX tables as provided may be in breach of your local RF laws. It is up to the end user to research and comply with local regulations and in using these presets the user assumes all liability associated with breaching local regulations.
```
Comment (KarateBrot) [inline presets/4.3/vtx/HDZero.txt:18]:
```suggestion
#$ PARSER: MARKED

#$ DESCRIPTION: 
#$ DESCRIPTION: <img src="https://static.wixstatic.com/media/967e02_82348b3d176249aea0926d07a2de0257~mv2_d_5559_3125_s_4_2.png" width="250px" style="margin-left: auto; margin-right: auto; display: block;"/>
#$ DESCRIPTION: 
#$ DESCRIPTION: # VTX tables and Canvas mode setup
#$ DESCRIPTION: ## for all HDzero digital systems
#$ DESCRIPTION: <br>
#$ DESCRIPTION: The information provided on this preset is for educational and entertainment purposes only. Betaflight makes no representations as to the safety or legality of the use of any information provided herein. End users assume all responsibility and liability for ensuring they are complying with all relevant laws and regulations.
#$ DESCRIPTION: <br><br>
#$ DESCRIPTION: Using the VTX tables as provided may be in breach of your local RF laws. It is up to the end user to research and comply with local regulations and in using these presets the user assumes all liability associated with breaching local regulations.
```

**Review by KarateBrot — CHANGES_REQUESTED** [review]

**Review by limonspb — CHANGES_REQUESTED** [review]
please don't forget to make 1 commit.
Just ALWAYS amend, cmon :)

**Comment by sugaarK** [comment]
karate was making the changes not me... ive squashed em now

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**